english/time-travel/0/7.md

```diff
@@ -0,0 +1,8 @@
+**You are looking around and notice strange-looking mushrooms 🍄.**
+
+When you look at mushrooms, you hear animal sounds.
+
+- [🍄 Eat mushrooms](7-1.md) 
```

Comment:
When I click on 'Eat Mushrooms', I am redirected to 7-1.md.
These 2 feels disconnected to me as there is no mention about eating mushrooms.

Reply:
Here I meant that after you ate mushrooms and world started to spinning and you are high